AUTHORS.md

```diff
@@ -0,0 +1,11 @@
+# Authors of Vivarium Core
+
+Vivarium Core was built by:
+
+* Members of the [Covert Lab](https://covert.stanford.edu) at
+  [Stanford University](https://stanford.edu):
+
+  * [Eran Agmon](https://eagmon.tumblr.com/home): `eagmon AT stanford DOT edu`
+  * Ryan Spangler: `ryan.spangler AT gmail DOT com`
+  * [Chris Skalnik](https://u8nwxd.github.io)
+  * Jerry Morrison: `j.erry.morrison AT gmail DOT com`
```

Comment:
jerry.morrison

Reply:
This is actually how Jerry asked that his email be specified. Gmail ignores dots when resolving email addresses, so he'll still get emails to this address. However, if a spammer gets this address, Jerry can tell Gmail to delete emails to `j.erry.morrison@...` to block the spam. As a side note, Gmail also ignores everything after `+` but before `@` so you can do things like this: `example+<anything you like>@gmail.com`